docs/getting-started/quickstart.md

```diff
@@ -3,160 +3,247 @@ title: Prefect Quickstart
-    """Get info about a repo - will retry twice after failing"""
-    url = f"https://api.github.com/repos/{repo_owner}/{repo_name}"
-    api_response = httpx.get(url)
-    api_response.raise_for_status()
-    repo_info = api_response.json()
-    return repo_info
+@flow(log_prints=True)
+def get_repo_info():
+    url = "https://api.github.com/repos/PrefectHQ/prefect"
+    response = httpx.get(url)
+    repo = response.json()
+    print("PrefectHQ/prefect repository statistics 🤓:")
+    print(f"Stars 🌠 : {repo['stargazers_count']}")
 
+if __name__ == "__main__":
+    get_repo_info()
+```
 
-@task
-def get_contributors(repo_info: dict):
-    contributors_url = repo_info["contributors_url"]
-    response = httpx.get(contributors_url)
-    response.raise_for_status()
-    contributors = response.json()
-    return contributors
+Note that we added a `log_prints=True` argument to the `@flow` decorator so that `print` statements within the flow-decorated function will be logged.
 
+<div class="terminal">
 
-@flow(name="Repo Info", log_prints=True)
-def repo_info(repo_owner: str = "PrefectHQ", repo_name: str = "prefect"):
-    """
-    Given a GitHub repository, logs the number of stargazers
-    and contributors for that repo.
-    """
-    repo_info = get_repo_info(repo_owner, repo_name)
-    print(f"Stars 🌠 : {repo_info['stargazers_count']}")
+```bash
+python my_workflow.py
+```
 
-    contributors = get_contributors(repo_info)
-    print(f"Number of contributors 👷: {len(contributors)}")
+</div>
 
+Now when we run this script, Prefect will automatically track the state of the flow run and log the output where we can see it in the UI or CLI.
 
-if __name__ == "__main__":
-    # create your first deployment
-    repo_info.serve(name="my-first-deployment")
+<div class="terminal">
+
+```bash
+15:16:34.705 | INFO    | prefect.engine - Created flow run 'frisky-reindeer' for flow 'get-repo-info'
+15:16:34.706 | INFO    | Flow run 'frisky-reindeer' - View at https://app.prefect.cloud/account/abc/flow-runs/flow-run/123
+15:16:35.053 | INFO    | Flow run 'frisky-reindeer' - PrefectHQ/prefect repository statistics 🤓:
+15:16:35.055 | INFO    | Flow run 'frisky-reindeer' - Stars 🌠 : 13593
+15:16:35.744 | INFO    | Flow run 'frisky-reindeer' - Finished in state Completed()
 ```
 
-Notice that we can write standard Python code within our flow _or_ break it down into component tasks, depending on the level of control and observability we want.
+</div>
+
+You should see similar output in your terminal, with your own randomly generated flow run name and your own Prefect Cloud account URL.
+
+## Step 4: Choose a remote infrastructure location
+
+Let's get this workflow running on infrastructure other than your local machine!
+We can tell Prefect where we want to run our workflow by creating a [work pool](/concepts/work-pools/).
+
+Because we're using Prefect Cloud, we have access to Prefect Managed work pools that provides hosted infrastructure for running our flows.
```

Comment:
```suggestion
Because we're using Prefect Cloud, we have access to Prefect Managed work pools that provide hosted infrastructure for running our flows.
```